sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+inherit python-single-r1
+
+GITHUB_USER="fearedbliss"
+GITHUB_REPO="bliss-initramfs"
+GITHUB_TAG="${PV}"
+
+DESCRIPTION="Boot your system's rootfs from OpenZFS/LUKS."
```

Comment:
you do not want to have dot in the end of the `DESCRIPTION` (minor nitpick, yes)

Reply:
fixing